experimenter/experimenter/glean/templates/glean/opt_out_alert.html

```diff
@@ -11,8 +11,8 @@
       <ul class="nav">
         <li class="nav-item my-2 ms-2">
           Nimbus is now tracking user interactions with Glean! You can opt-out of this
-          data collection by clicking "Glean Opt-Out" in the top right. For more
-          information see our
+          data collection by clicking "Glean Opt-In" under the settings drop down
```

Comment:
```suggestion
          data collection by clicking "Glean Opt-Out" under the settings drop down
```

Reply:
@relud I switched it to use a toggle to match the style of the the theme toggle, so in this case, toggle enabled is glean enabled, and toggle disabled is glean disabled, so i updated the wording on the switch and in the alert?